lib/browser/compiler/core.js

```diff
@@ -103,9 +103,9 @@ riot.compile = function(arg, fn) {
 // reassign mount methods
 var mount = riot.mount
 
-riot.mount = function(a, b, c) {
+riot.mount = function(a, b, c, d) {
```

Comment:
I like the idea, and I agree, the return value is currently not very meaningful, but I think if implemented, should include all signatrues 

ex. `riot.mount('*', function(ret) {...})` etc

http://riotjs.com/api/#mounting

but I am ok with just sticking with the following, keeps the api more simple

```javascript
riot.compile(function() {
  var tags = riot.mount('*')
  console.log(tags)
})
```